main.py

```diff
@@ -161,6 +162,7 @@ def wrapper(*args, **kwargs):
 @input_error
 def add_contact(args, book: AddressBook):
     name, phone = args[0], args[1]
+    email = args[2] if len(args) > 2 else None  # Додаємо email, якщо він вказаний
```

Comment:
no need to make email as a part of "add-contact" command
let's use "add-email" just as a separate command